pycodestyle.py

```diff
@@ -1554,6 +1561,8 @@ def check_logical(self):
         if self.logical_line:
             self.previous_indent_level = self.indent_level
             self.previous_logical = self.logical_line
+            if not self.indent_level:
+                self.previous_logical_toplevel = self.logical_line
```

Comment:
I've thought about it, I'm not sure what alternative variable name to use. Open to suggestions!